### PR TITLE
layout changed for home page

### PR DESCRIPTION
**Pull request by Kratika0907**
Layout changed for home page
Filter logic removed from main component
Link added for shop and cart

### CODE REVIEW

File: components/coins-status/index.js [hunk 1]
```diff
@@ -47,10 +46,8 @@ const CoinsStatus = (props) => {
           .coins-container {
             display: flex;
             flex-direction: row;
-            justify-content: space-evenly;
-            align-self: flex-end;
-            width: 33%;
-            flex-wrap: nowrap;
+            //justify-content: space-evenly;
```
Comment (swarajpure) [inline components/coins-status/index.js:49]: Commented code... Are you sure if we need this?

File: styles/Home.module.css [hunk 2]
```diff
@@ -13,23 +13,23 @@
   --dark-bronze: #ae7235;
 }
 
-.header {
+/* .header {
```
Comment (swarajpure) [inline styles/Home.module.css:16]: One more commented code..

**Review by swarajpure — COMMENTED** [review]
Reviewed for the commented code.

**Review by sumitd94 — APPROVED** [review]

File: styles/Home.module.css [hunk 3]
```diff
@@ -146,3 +146,100 @@
...
+
+
+
+	.homeHeader{
+		grid-column: 1 / 4;
+    grid-row: 1;
+	}
+
+	.homeLogin {
+    grid-column: 4 ;
```
Comment (whyDontI) [inline styles/Home.module.css:209]: Fix indentation for this file. Also please remove redundant blank lines if not removed by linter itself.

File: pages/index.js [hunk 4]
```diff
@@ -1,82 +1,85 @@
...
-          handleDebitSort={handleDebitSort}
-          showOriginal={showOriginal}
-        />
-        <TransactionList transactions={transaction} />
       </div>
-      <div className={`${styles.card}`}>
+
+      <div className={styles.homeTransaction}>
+        <div className={styles.transactionGraph}>
+ <div className={`${styles.card}`}>
```
Comment (whyDontI) [inline pages/index.js:62]: Indentation needs to be done here

**Review by whyDontI — CHANGES_REQUESTED** [review]

**Pull request closed without merging**